js/interface.js

```diff
@@ -70,7 +72,7 @@ function getDataSources() {
   $('#trash-sources').hide();
 
   return Fliplet.DataSources.get({
-    appId: isShowingAll ? undefined : copyData.appId,
```

Comment:
Please explain this change

Reply:
@squallstar the change was made only to keep consistent, as we are using similar syntax in showTrashedDataSources. As I tested it doesn't change any result from the API. Should we use undefined instead?

Reply:
Please use `undefined` if the result is the same